ambersim/utils/_internal_utils.py

```diff
@@ -4,7 +4,7 @@
 from ambersim import ROOT
 
 
-def _check_filepath(filepath: Union[str, Path]) -> str:
```

Comment:
Example: `pyright` complains about `_check_filepath` because the leading underscore implies it's a private function of the current file, but it's unused within the file. Further, I import this elsewhere, so it's clearly not private.